22:46:08,140 WARN  eu.stratosphere.nephele.configuration.GlobalConfiguration     - Cannot load configuration: unknown element plugins
22:46:08,177 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Reading location of job manager from configuration
22:46:08,188 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Determined address of job manager to be cloud-11.dima.tu-berlin.de/130.149.21.15:6002
22:46:08,198 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Announcing connection information cloud-12 to job manager
22:46:08,270 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server Responder: starting
22:46:08,270 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server listener on 6122: starting
22:46:08,271 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server handler 0 on 6122: starting
22:46:09,377 INFO  eu.stratosphere.nephele.taskmanager.bufferprovider.GlobalBufferPool  - Initialized global buffer pool with 32768 buffers with a size 32768 bytes each
22:46:09,397 INFO  eu.stratosphere.nephele.taskmanager.bytebuffered.ByteBufferedChannelManager  - Initialized byte buffered channel manager with sender-side spilling disabled
22:46:09,407 INFO  eu.stratosphere.nephele.instance.HardwareDescriptionFactory   - Found Tenured Gen pool (max: 24051843072, used: 0)
22:46:09,407 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Initializing memory manager with 22528 megabytes of memory
22:46:46,254 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Build Triads (7/25)
22:46:46,258 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Build Triads (7/25)
22:46:47,694 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (55/200)
22:46:47,695 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (55/200)
22:46:47,697 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (55/200)
22:46:47,698 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (55/200)
22:46:48,352 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (54/200)
22:46:48,352 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (54/200)
22:46:48,354 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (54/200)
22:46:48,355 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (54/200)
22:46:48,743 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (56/200)
22:46:48,743 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (56/200)
22:46:48,744 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (56/200)
22:46:48,745 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (56/200)
22:46:49,375 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (53/200)
22:46:49,375 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (53/200)
22:46:49,377 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (53/200)
22:46:49,378 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (53/200)
22:46:50,303 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (51/200)
22:46:50,304 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (51/200)
22:46:50,306 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (51/200)
22:46:50,307 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (51/200)
22:46:51,298 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (49/200)
22:46:51,298 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (49/200)
22:46:51,300 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (49/200)
22:46:51,301 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (49/200)
22:46:52,103 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (52/200)
22:46:52,103 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (52/200)
22:46:52,105 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (52/200)
22:46:52,106 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (52/200)
22:46:52,508 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (50/200)
22:46:52,508 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (50/200)
22:46:52,510 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (50/200)
22:46:52,512 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (50/200)
22:47:22,577 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Finished PACT code.: Build Triads (7/25)
22:47:22,577 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from RUNNING to FINISHING for task Build Triads (7/25)
22:49:25,396 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-952704198313740508_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:47483 remote=/130.149.21.16:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:25,397 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-952704198313740508_1722 bad datanode[0] 130.149.21.16:45020
22:49:25,398 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-952704198313740508_1722 in pipeline 130.149.21.16:45020, 130.149.21.66:45020, 130.149.21.21:45020: bad datanode 130.149.21.16:45020
22:49:28,005 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7657255860791682511_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:47485 remote=/130.149.21.16:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:28,006 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7657255860791682511_1722 bad datanode[0] 130.149.21.16:45020
22:49:28,006 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7657255860791682511_1722 in pipeline 130.149.21.16:45020, 130.149.21.28:45020, 130.149.21.34:45020: bad datanode 130.149.21.16:45020
22:49:31,179 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_6960913453043916879_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:47480 remote=/130.149.21.16:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:31,180 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_6960913453043916879_1722 bad datanode[0] 130.149.21.16:45020
22:49:31,181 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_6960913453043916879_1722 in pipeline 130.149.21.16:45020, 130.149.21.33:45020, 130.149.21.17:45020: bad datanode 130.149.21.16:45020
22:49:35,006 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-6590610836056644199_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:47476 remote=/130.149.21.16:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:35,007 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-6590610836056644199_1722 bad datanode[0] 130.149.21.16:45020
22:49:35,008 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-6590610836056644199_1722 in pipeline 130.149.21.16:45020, 130.149.21.24:45020, 130.149.21.61:45020: bad datanode 130.149.21.16:45020
22:49:37,608 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_42010787610343374_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:47497 remote=/130.149.21.16:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:37,609 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_42010787610343374_1722 bad datanode[0] 130.149.21.16:45020
22:49:37,610 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_42010787610343374_1722 in pipeline 130.149.21.16:45020, 130.149.21.23:45020, 130.149.21.60:45020: bad datanode 130.149.21.16:45020
22:49:42,616 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_1959115210794619274_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:47499 remote=/130.149.21.16:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:42,617 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1959115210794619274_1722 bad datanode[0] 130.149.21.16:45020
22:49:42,618 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1959115210794619274_1722 in pipeline 130.149.21.16:45020, 130.149.21.2:45020, 130.149.21.3:45020: bad datanode 130.149.21.16:45020
22:49:52,871 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-4739951749937144072_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:47487 remote=/130.149.21.16:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:52,872 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-4739951749937144072_1722 bad datanode[0] 130.149.21.16:45020
22:49:52,872 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-4739951749937144072_1722 in pipeline 130.149.21.16:45020, 130.149.21.2:45020, 130.149.21.66:45020: bad datanode 130.149.21.16:45020
22:50:14,896 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-970868090595704934_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:47494 remote=/130.149.21.16:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:14,896 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-970868090595704934_1722 bad datanode[0] 130.149.21.16:45020
22:50:14,897 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-970868090595704934_1722 in pipeline 130.149.21.16:45020, 130.149.21.21:45020, 130.149.21.65:45020: bad datanode 130.149.21.16:45020
22:50:31,544 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-952704198313740508_1742java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:38660 remote=/130.149.21.66:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:31,544 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-952704198313740508_1742 bad datanode[0] 130.149.21.66:45020
22:50:31,545 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-952704198313740508_1742 in pipeline 130.149.21.66:45020, 130.149.21.21:45020: bad datanode 130.149.21.66:45020
22:50:34,106 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7657255860791682511_1755java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:45545 remote=/130.149.21.28:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:34,107 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7657255860791682511_1755 bad datanode[0] 130.149.21.28:45020
22:50:34,107 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7657255860791682511_1755 in pipeline 130.149.21.28:45020, 130.149.21.34:45020: bad datanode 130.149.21.28:45020
22:50:37,280 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_6960913453043916879_1775java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:42644 remote=/130.149.21.33:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:37,282 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_6960913453043916879_1775 bad datanode[0] 130.149.21.33:45020
22:50:37,282 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_6960913453043916879_1775 in pipeline 130.149.21.33:45020, 130.149.21.17:45020: bad datanode 130.149.21.33:45020
22:50:41,105 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-6590610836056644199_1782java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:43682 remote=/130.149.21.24:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:41,106 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-6590610836056644199_1782 bad datanode[0] 130.149.21.24:45020
22:50:41,107 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-6590610836056644199_1782 in pipeline 130.149.21.24:45020, 130.149.21.61:45020: bad datanode 130.149.21.24:45020
22:50:43,706 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_42010787610343374_1785java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:36814 remote=/130.149.21.23:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:43,708 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_42010787610343374_1785 bad datanode[0] 130.149.21.23:45020
22:50:43,708 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_42010787610343374_1785 in pipeline 130.149.21.23:45020, 130.149.21.60:45020: bad datanode 130.149.21.23:45020
22:50:48,727 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_1959115210794619274_1791java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:43739 remote=/130.149.21.2:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:48,728 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1959115210794619274_1791 bad datanode[0] 130.149.21.2:45020
22:50:48,729 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1959115210794619274_1791 in pipeline 130.149.21.2:45020, 130.149.21.3:45020: bad datanode 130.149.21.2:45020
22:50:58,968 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-4739951749937144072_1811java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:43742 remote=/130.149.21.2:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:58,969 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-4739951749937144072_1811 bad datanode[0] 130.149.21.2:45020
22:50:58,970 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-4739951749937144072_1811 in pipeline 130.149.21.2:45020, 130.149.21.66:45020: bad datanode 130.149.21.2:45020
22:51:20,995 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-970868090595704934_1871java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.16:35059 remote=/130.149.21.21:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:20,997 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-970868090595704934_1871 bad datanode[0] 130.149.21.21:45020
22:51:20,997 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-970868090595704934_1871 in pipeline 130.149.21.21:45020, 130.149.21.65:45020: bad datanode 130.149.21.21:45020
